### PR TITLE
fixed footer position

### PR DESCRIPTION
**Pull request by MorganeL-pro**
Fixed the pb with footer display on login page.

### CODE REVIEW

**Review by Carolinecrepin — APPROVED** [review]
it's good ;)

**Review by Carolinecrepin — DISMISSED** [review]
ok good :)

File: templates/base.html.twig [hunk 1]
```diff
@@ -17,6 +17,11 @@
 		{% endblock %}
 	</head>
 	<body>
-		{% block body %}{% endblock %}
+		<main>
+			<div class="container">
+				{% block body %}{% endblock %}
+			</div>
+			{% include 'components/_footer.html.twig' %}
+		</main>
```
Comment (RomainClair) [inline templates/base.html.twig:25]: Isn't it strange to have the footer in the main ?
You could have a main (and skip the div) and a footer after it

**Review by RomainClair — COMMENTED** [review]
I don't clearly see why you have a _footer components as it will probably only be included once in the base...

**Pull request closed without merging**